libs/wingc/src/jsify/snapshots/no_capture_of_identifier_from_inner_scope.snap

```diff
@@ -52,7 +52,7 @@ class $Root extends $stdlib.std.Resource {
   constructor($scope, $id) {
     super($scope, $id);
     class $Closure1 extends $stdlib.std.Resource {
-      _hash = $stdlib.core.closureId();
+      _id = $stdlib.core.closureId();
```

Comment:
tip: just to avoid this change polluting your diff you could submit a quick PR to just change this and then rebase.